ui/src/literature/components/__tests__/CiteAllAction.test.jsx

```diff
@@ -33,12 +36,32 @@ describe('CiteAllAction', () => {
   it('calls OnCiteClick when option clicked', () => {
```

Comment:
this test is redundant, but I guess we can keep it for now, until the other is fixed